packages/react-storage/src/components/StorageBrowser/controls/hooks/usePagination.ts

```diff
@@ -0,0 +1,31 @@
+import { PaginationProps } from '../../composables/Pagination';
+import { useControlsContext } from '../context';
+
+export type UsePagination = () => {
+  props?: PaginationProps;
+};
+
+export const usePagination: UsePagination = () => {
```

Comment:
And instead use `PaginationProps | null` as the return type

Reply:
Done